src/main/java/com/softserve/sprintone/jose/TaskJoseSprintOne.java

```diff
@@ -1,11 +1,9 @@
 package com.softserve.sprintone.jose;
 
 import com.softserve.sprintone.exception.TaskException;
-
 import java.io.BufferedReader;
 import java.io.IOException;
-import java.util.InputMismatchException;
-import java.util.Scanner;
+import java.util.*;
```

Comment:
do not use start for importing an entire package.

Be careful when doing changes over classes from peers. Why are you committing changes over @JoseCaste 's class?

Reply:
sorry, this was an accident when I did a pull from dev, can you help me or give me a hint to resolve this?, thank you